anhalyze/core/anhalyze.py

```diff
@@ -219,13 +224,17 @@ def _init_range(self):
         # Init grid geocoordinates range
         self.attrs['coord_lat_range'] = [np.nanmin(lat), np.nanmax(lat)]
         self.attrs['coord_lon_range'] = [np.nanmin(lon), np.nanmax(lon)]
-        self.attrs['coord_depth_range'] = [self.coords[self.attrs['coord_depth']].data.min(),
-                                           self.coords[self.attrs['coord_depth']].data.max()]
-
+        
         # Init grid dims range
         self.attrs['dim_x_range'] = [0, self._xr_dataset.sizes[self.attrs['dim_x']]]
         self.attrs['dim_y_range'] = [0, self._xr_dataset.sizes[self.attrs['dim_y']]]
-        self.attrs['dim_z_range'] = [0, self._xr_dataset.sizes[self.attrs['dim_z']]]
+        
+        # Only for 3 dimension variable
+        if hasattr(self, 'coord_depth'):
```

Comment:
It is crashing for me if I try to open a gridT file and then make a selection. 

This is the code that I used.
```
aa = ah.AnhaDataset('./Test_Data/ANHA4-EPM111_y1998m08d03_gridT.nc')
aaa = aa.sel([65,50],[-93,-75],[0,300])
```

For me the bug is related to this line. It is returning `False` for me.
The thing is that 'coord_depth' is not an `attr` of `self`. It is a key in the `attrs` dictionary. Maybe a bit confusing, but `attrs` is an `attr` of `self`. 
If you are getting the same error, please replace the line with:
```
if 'coord_depth' in aa.attrs:
```